src/main/java/at/hannibal2/skyhanni/utils/UtilsPatterns.kt

```diff
@@ -0,0 +1,21 @@
+package at.hannibal2.skyhanni.utils
+
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+
+object UtilsPatterns {
+    private val patternGroup = RepoPattern.group("utils")
+
+    /** Examples:
+    §d§l§ka§r §d§l§d§lMYTHIC ACCESSORY §d§l§ka
+    §d§l§ka§r §d§l§d§lSHINY MYTHIC DUNGEON CHESTPLATE §d§l§ka
+    §6§lSHINY LEGENDARY DUNGEON BOOTS
+    §6§lLEGENDARY DUNGEON BOOTS
+    §5§lEPIC BOOTS
+    §f§lCOMMON
+     **/
+    val rarityLoreLinePattern by patternGroup.pattern("item.lore.rarity.line", "^(?:§.){2,3}(?:.§. (?:§.){4})?(?:SHINY )?(?<Rarity>${LorenzRarity.entries.joinToString(separator = "|") { it.rawName }}) ?(?:DUNGEON )?(?<ItemCategory>[^§]*)(?: (?:§.){3}.)?$")
```

Comment:
maybe this could be done in a separate function `${LorenzRarity.entries.joinToString(separator = "|") { it.rawName }}` but not super necessary